src/plugins/plugin.legend.js

```diff
@@ -408,19 +417,30 @@ var Legend = Element.extend({
 				}
 			};
 
+			var alignmentOffset = function(dimension, blockSize) {
+				if (opts.align === 'start') {
+					return labelOpts.padding;
+				} else if (opts.align === 'end') {
```

Comment:
The `else-if` is necessary here since the last case is treated as the default/unrecognized case (`center`) only if checks against the two recognized options fails (`start` and `end`).  I originally did have a final dangling `else` block for `center`, which the linter did catch/force me to remove.

Reply:
Except if I missed something obvious, this is equivalent to:

```javascript
if (opts.align === 'start') {
    return labelOpts.padding;
}
if (opts.align === 'end') {
    return dimension - blockSize;
}
// default to center
return (dimension - blockSize + labelOpts.padding) / 2;
```

ESLint should complain about the `} else if (...) {` since there is `return` in all blocks.

could also be:

```javascript
switch (opts.align) {
case 'start':
    return labelOpts.padding;
case 'end':
    return dimension - blockSize;
default: // center
    return (dimension - blockSize + labelOpts.padding) / 2;
}
```

Reply:
Ahh - yes, you are correct.  And in fact, I do prefer the `switch` syntax anyhow - it's clearer, with no variations on how it could be structured.  Will roll that into the next commit.